docs/preview/features/ps/keyvault.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Scripts related to interacting with Azure Key Vault"
+layout: default
+---
+
+# Azure Key Vault scripting
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Import-Module -Name Arcus.Scripting.PS.KeyVault
```

Comment:
Is there a reason why we have `PS` in here? Do we support others?

Reply:
Currently: no
This was included to ensure the repo would be future-proof and while those might not end up on the PS-gallery, seems better to allign naming conventions?
Related to #23

Reply:
Yes, I think it's best to leave it out indeed as different scripting might require different package managers indeed

Reply:
Moved `/ps` to `/powershell`.